python/ray/_private/gpu_object_manager.py

```diff
@@ -43,12 +43,12 @@ def __init__(self):
         #
         # Note: Currently, `gpu_object_store` is only supported for Ray Actors.
         self.gpu_object_store: Dict[str, List["torch.Tensor"]] = {}
-        # A dictionary that maps from owned object ref to a metadata tuple: (actor handle, object ref).
+        # A dictionary that maps from owned object ID to a metadata tuple: (actor handle, object ref).
```

Comment:
Can we key on ObjectID instead of hex str?

Reply:
Would you mind sharing why you prefer using an `ObjectID` rather than a hex string?